config/swagger.yaml

```diff
@@ -0,0 +1,140 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Webhook router config
+  license:
+    name: MIT
+host: 127.0.0.1:8081
+paths:
+  /routes/{token}:
+    parameters:
+      - name: token
+        in: path
+        required: true
+        description: The token of the route
+        type: string
+    delete:
+      summary: Delete the given webhook
+      operationId: delete_route
+      responses:
+        '204':
+          description: Webhook routed correctly
+        default:
+          description: unexpected error
+          schema:
+            $ref: "#/definitions/Error"
+    get:
+      summary: Gets the configuration for the given webhook
+      operationId: get_route
+      responses:
+        '200':
+          description: The correct webhook
+          schema:
+            $ref: "#/definitions/Route"
+        default:
+          description: unexpected error
+          schema:
+            $ref: "#/definitions/Error"
+    patch:
+      summary: Edit the configuration of the given webhook
+      operationId: patch_route
+      parameters:
+        - name: new_info
+          in: body
+          required: true
+          schema:
+            $ref: "#/definitions/PatchRoute"
+      responses:
+        '204':
+          description: Routing patched correctly
+        default:
+          description: unexpected error
+          schema:
+            $ref: "#/definitions/Error"
+  /routes/{token}/regenerate:
+    parameters:
+      - name: token
+        in: path
+        required: true
+        description: The token of the route
+        type: string
+    post:
+      summary: Regenerates the token for the given route
+      operationId: regenerate_token
+      responses:
+        '200':
+          description: Successful regeneration, returns json data of the new route
+          schema:
+            $ref: "#/definitions/Route"
+        default:
+          description: unexpected error
+          schema:
+            $ref: "#/definitions/Error"
+  /routes:
+    get:
+      summary: Gets all routes that are configured for given user
+      operationId: get_all_routes
+      responses:
+        '200':
+          description: Returns the given routes
+          schema:
+            $ref: "#/definitions/Routes"
+        default:
+          description: unexpected error
+          schema:
+            $ref: "#/definitions/Error"
+  /add-route:
+    post:
+      summary: Gets all routes that are configured
+      operationId: add_route
+      parameters:
+        - name: new_route
+          in: body
+          required: true
+          schema:
+            $ref: "#/definitions/NewRoute"
+      responses:
+        '201':
+          description: Expected response to a valid request
+          schema:
+            $ref: "#/definitions/Route"
+        default:
+          description: unexpected error
+          schema:
+            $ref: "#/definitions/Error"
+definitions:
+  NewRoute:
```

Comment:
It's a bit unclear from the names what the difference between a `NewRoute` and a `Route` is. Is a tokenless route still a route or is it something else?